elastic/kibana
# [App Search] API logs: Server route + ApiLogsLogic + useEffects

## Summary

3 main goals of this PR:

1. Server route: https://github.com/elastic/kibana/commit/b52ae2c9089e7c68131975d94a1a899ae73e3573
2. ApiLogsLogic: https://github.com/elastic/kibana/commit/3d772320c56f45aeadcf155a4138bd83e45b3a90, https://github.com/elastic/kibana/commit/389096d4ba4b4b76b822f029f44398bdfa81e09b, https://github.com/elastic/kibana/commit/2ae83ba6871d1ab134a04e8126d92c75b6905284
3. View useEffects: https://github.com/elastic/kibana/commit/2250d8228d4466dbf5df249583033a57d04d1f9c

## QA

This PR doesn't have any UI changes yet (next set of PRs!) but does add polling behavior that you can QA:

<img width="1321" alt="" src="https://user-images.githubusercontent.com/549407/112919168-63a56580-90bb-11eb-9445-9890b21e3b47.png">

- Go to any engine's API logs:
  - [x] Confirm an `api_logs` call fires on load
  - [x] Confirm an `api_logs` call fires every **5** seconds after that
  - (TODO in future PR) - Confirm that paginating correctly calls an `api_logs` call and updates the table
- Go to the engine overview:
  - [x] Confirm an `api_logs` call fires on load
  - [x] Confirm an `api_logs` call fires every **5** seconds after that

### Checklist
